chromium_src/components/policy/resources/policy_templates.py

```diff
@@ -0,0 +1,124 @@
+    # This is where we need to inject the Brave specific names. The policies
+    # themselves are already defined (under `policy_definitions`), we just need
+    # to add a mapping for ID (integer; unique) and name (matches name under
+    # `policy_definitions`).
+    #
+    #
+    # There are some other fields which are not used by this script:
+    # - "common_schemas"
+    # - "legacy_device_policy_proto_map"
+    # - "manual_device_policy_proto_map"
+    # - "risk_tag_definitions"
+
+    policy_yaml = policies['policies']
+    policy_section = policy_yaml['policies']
+
+    # get the highest ID in the file
+    highest_number = 0
+    for key, _ in policy_section.items():
+        if int(key) > highest_number:
+            highest_number = int(key)
+
+    # append our entries to the ones from policies.yaml
+    # TODO(bsclifton): we can create this array dynamically by reading the file
+    # names from:
+    # `//brave/components/policy/resources/templates/policy_definitions/BraveSoftware` # pylint: disable=line-too-long
+    brave_policies = [
+        'TorDisabled', 'BraveRewardsDisabled', 'BraveWalletDisabled',
+        'BraveVPNDisabled', 'BraveAIChatEnabled', 'BraveSyncUrl',
+        'BraveShieldsDisabledForUrls', 'BraveShieldsEnabledForUrls'
+    ]
+    for entry in brave_policies:
+        highest_number += 1
+        #policy_key = str(highest_number)
+        policy_section[highest_number] = entry
+
+    return policies
+
+
+def update_policy_files():
+    # Chromium stores all group policy definitions under
+    # `//components/policy/resources/templates/policy_definitions/`
+    #
+    # The name of the file (minus the extension; ex: TorDisable.yaml => TorDisable)
+    # corresponds to an auto-generated entry in:
+    # `//out/<build_type_here>/gen/chrome/app/policy/policy_templates.json
+    #
+    # That auto-generated value (ex: `policy::key::kTorDisabled`) is referenced
+    # when we map to a preference in our policy map:
+    # `//brave/browser/policy/brave_simple_policy_map.h`
+    #
+    # When the code below is ran this will copy the group policy files from:
+    # `//brave/components/policy/resources/templates/policy_definitions/`
+    # to their expected place in Chromium:
+    # `//components/policy/resources/templates/policy_definitions/`
+    #
+    policy_dir = wspath(
+        "//brave/components/policy/resources/templates/policy_definitions/")
+    with os.scandir(policy_dir) as entries:
+        for entry in entries:
+            if not entry.is_dir():
+                continue
+            src_dir = entry.path
+            src_dir_name = entry.name
+            dst_dir = wspath(
+                f"//components/policy/resources/templates/policy_definitions/{src_dir_name}"  # pylint: disable=line-too-long
+            )
+            shutil.copytree(src_dir,
+                            dst_dir,
+                            dirs_exist_ok=True,
+                            copy_function=copy_only_if_modified)
```

Comment:
```suggestion
    with os.scandir(policy_dir) as entries:
        for entry in entries:
            if not entry.is_dir():
                continue
            dst_dir = wspath(
                f"//components/policy/resources/templates/policy_definitions/" +
                entry.name
            )
            shutil.copytree(entry.path,
                            dst_dir,
                            dirs_exist_ok=True,
                            copy_function=copy_only_if_modified)
```

Reply:
Also, I believe the present implementation does not handle the case where one of our `.yaml` policy files gets deleted.